DXMainClient/Online/CnCNetManager.cs

```diff
@@ -770,6 +777,13 @@ private void DoUserListReceived(string channelName, string[] userList)
             MultipleUsersAdded?.Invoke(this, EventArgs.Empty);
 
             channel.OnUserListReceived(channelUserList);
+
+            // We only need to request user info once, and chat channels only.
```

Comment:
Sorry, but could you explain why this user list is initialized for only once? I mean, what if a new user joins?

Reply:
When a user joins the lobby, a who is done and we aquire the info. We need it just once when joining for everyone.

Reply:
uh... suppose 6 players join a room (I don't know if 'who' is sent in the lobby or a room), how many 'who' (what kind of information does it query for) query is sent, and how many list is created?

Also, is such a message periodically updated, or not?

Reply:
Actually, where did the who query go?

Reply:
https://github.com/CnCNet/xna-cncnet-client/pull/651/files#diff-611aadbae7dba9b618e463d4357561d7dfd7b15dfbf9e3cf7043d2ebc1271e68R779-R783


https://github.com/CnCNet/xna-cncnet-client/blob/3d36caff78f28c61428b407f4d463d6c9aef5cb9/DXMainClient/Online/Channel.cs#L326-L329